octree_web_viewer/src/bin/points_web_viewer.rs

```diff
@@ -69,24 +67,44 @@ fn main() {
         ])
         .get_matches();
 
-    let port = value_t!(matches, "port", u16).unwrap_or(5433);
+    let port = value_t!(matches, "port", u16).unwrap();
+    let ip_port = format!("127.0.0.1:{}", port);
     let octree_directory = PathBuf::from(matches.value_of("octree_directory").unwrap());
 
-    let octree = {
-        let octree = match octree::OnDiskOctree::new(octree_directory) {
-            Ok(octree) => octree,
+    // CAVEAT: this actix-web framework handles requests asynchronously.
+    // For multithreading with tree updates consider using the actix actor system.
+    let octree: Arc<dyn octree::Octree> = {
+        let my_octree = match octree::OnDiskOctree::new(octree_directory) {
```

Comment:
Nit: rename all occurrences of `my_octree` with `octree`.

Reply:
the compiler complains if i put octree inside the brackets, i put web_octree instead (should be already versioned)